### ISSUE TITLE
How to check that we have property 'a' and it equals to 'value'?

### ISSUE THREAD

**Issue by hyzhak** [astorije/chai-immutable#41]
To have something like original does

```javascript
expect(Map({'a': 'value'})).to.have.property('a', 'value');
```

but for immutable objects

**Comment by astorije** [comment]
See #9 and #14. Also, this will be added with #43.

**Comment by astorije** [comment]
This is now available in v1.6.0: https://github.com/astorije/chai-immutable/releases/tag/v1.6.0
